resources/puppetlabs/lein-ezbake/template/foss/ext/default.erb

```diff
@@ -2,8 +2,8 @@
 # Init settings for <%= EZBake::Config[:project] %>
 ###########################################
 
-# Location of your Java binary (version 8)
-JAVA_BIN="/usr/bin/java"
+# Location of your Java binary
+#JAVA_BIN=""
```

Comment:
I believe the CLI also uses this. There's places like https://github.com/OpenVoxProject/openvox-server/blob/da7e9916bcb456e0793d69a1e69e0edeb5bdfc05/resources/ext/cli_defaults/cli-defaults.sh.erb#L7 others in the puppetserver repo that use this.

When testing this, I think running `puppetserver --help` or `puppetserver ruby` should trigger any error if it isn't set.

Reply:
Related discussion: #5.